lib/redis-lock.rb

```diff
@@ -55,12 +55,16 @@ def unlock(force_remote = false)
       end
     end
 
+    def locked?
+      return !!@redis.get(@key)
```

Comment:
I'd prefer `!@redis.get(@key).nil?` and can you add a short comment consistent with the other methods in the file?